lib/e_chronicler/models/journal_entry.ex

```diff
@@ -42,4 +42,6 @@ defmodule EChronicler.Models.JournalEntry do
     |> Repo.insert()
   end
 
+  def get_journal_entry!(id), do: Repo.get!(JournalEntry, id)
```

Comment:
Should this function really fail passively when there's no entry with the given id? I'll note that you don't have a test for this case. You probably want to include that test case in both your model and controller tests.

What do you want to to tell the controller if there is no journal entry? And what should the controller do?

Reply:
I think it should display a particular error page, like a 404 that says "entry not found." `Ecto.get!` throws an error and `Ecto.get` returns `nil` if nothing is found, not sure which one makes more sense to use